example/.flutter-plugins-dependencies

```diff
@@ -0,0 +1 @@
+{"_info":"// This is a generated file; do not edit or check into version control.","dependencyGraph":[{"name":"flutter_callkeep","dependencies":[]}]}
```

Comment:
> do not edit or check into version control

Can you add this file to the gitignore?

Reply:
Yes, added.

Reply:
The file here is still part of the PR, can you make sure it isn't?

Reply:
deleted the file now